appcues/src/main/java/com/appcues/analytics/AnalyticsTracker.kt

```diff
@@ -29,13 +29,17 @@ internal class AnalyticsTracker(
         }
     }
 
-    fun identify(properties: Map<String, Any>? = null) {
+    fun identify(properties: Map<String, Any>? = null, interactive: Boolean = true) {
         if (!analyticsPolicy.canIdentify()) return
 
         activityBuilder.identify(properties).let {
             updateAnalyticsFlow(IDENTIFY, false, it)
 
-            analyticsQueueProcessor.flushThenSend(it)
+            if (interactive) {
+                analyticsQueueProcessor.flushThenSend(it)
+            } else {
+                analyticsQueueProcessor.queue(it)
+            }
```

Comment:
we now support non-interactive `identify` calls - so that the new `_appcuesForm_` attributes can be merged in with the flow events and do not need to directly trigger another call to the `/qualify` endpoint.